sql/pgtap.sql.in

```diff
@@ -6864,6 +6876,18 @@ RETURNS TEXT AS $$
     SELECT _relcomp( $1, $2, NULL::text, '' );
 $$ LANGUAGE sql;
 
+-- set_eq( array, array, description )
+CREATE OR REPLACE FUNCTION set_eq(anyarray, anyarray, TEXT)
+RETURNS TEXT AS $$
+    SELECT ok($1 @> $2 AND $2 @> $1, $3);
```

Comment:
This seems superfluous. Why not just use `is(array1, array2)`?

Reply:
`is(a1, a2)` is order-dependent but the order of our arrays is not guaranteed. Is there something else that would suit, short of unnesting both sides?